ts/Core/Axis/ZAxis.ts

```diff
@@ -29,6 +33,11 @@ const {
     splat
 } = U;
 
+defaultOptions.zAxis = merge(xAxis, {
```

Comment:
This should go into a `compose` or otherwise default options change uncontrolled during imports.